### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by armandocomellas1**
### Hi there! This project is based on creating a UI list with webpack packages

- Use semantic HTML tags.
- Apply best practices in HTML code.
- Use webpack to bundle JavaScript.
- Learn how to use proper ES6 syntax.
- Use ES6 modules to write modular JavaScript.
- Use src and dist folders insde each project.
👍

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,50 +1,33 @@
...
+  storeData.getElementsByClassName('editable')[0].focus();
+  storeData.style.backgroundColor = 'rgb(231, 230, 177)';
+});
+
+const deleteElement = ((checkEvent) => {
+  let keyCode = checkEvent.split(' ')[1];
+  keyCode -= 1;
+  arrayList.splice(keyCode, 1);
+  document.getElementsByClassName(checkEvent)[0].remove();
+  localStorage.setItem('List', JSON.stringify(arrayList));
```
Comment (andersonlebon) [inline src/index.js:26]: - [ ] KIndly make sure your data is preserved when the page is refreched. Kindly follow this screencast

https://user-images.githubusercontent.com/65068771/184257481-63549984-f962-465c-8817-0bd1ad5aa4c6.mp4

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @armandocomellas1 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (armandocomellas1) [reply src/index.js:26]: Thank you now it is loaded the data from local storage

**Review by rustam-tolipov — APPROVED** [review]
## Hello @armandocomellas1👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**